lstchain/scripts/lstchain_data_muon_analysis.py

```diff
@@ -153,17 +153,17 @@ def main():
         print("Event {}. Number of pixels above 10 phe: {}".format(event_id,
                                                                   np.size(image[0][image[0] > 10.])))
         
-        if not tag_pix_thr(image): #default skipps pedestal and calibration events
+        if not tag_pix_thr(image[0]): #default skipps pedestal and calibration events
```

Comment:
Why did you change this back to `image[0]` while in the rest of the script you corrected to just `image`. In this script, does the image array still contain the two gains for each pixel?

Reply:
you are right @morcuended. I should apply here gain selection... 
by the way, for the data that you are analyzing in LP, are you applying the proper threshold for the gain selection? According to #200, we do not have a proper threshold due to the ctapipe bug

Reply:
If you have the threshold set to `4094`, then it means that the threshold is not properly applied.

For the muon script, I do not think it is a good idea to have a duplicated calibration for the regular data analysis and muon data analysis, therefore I expect the script `lstchain_data_muon_analysis.py` to be deprecated and run the analysis only on calibrated files. I'll mark this in a comment on the script as no-gain selection, but I'm not sure if it is worth it to solve it at the moment.

Reply:
These are the warning I'm getting:
```
WARNING: AstropyDeprecationWarning: channel selection is now performed at <= R1 calibration level [lstchain.reco.dl0_to_dl1]
WARNING: AstropyDeprecationWarning: gain selection is now performed at <= R1 calibration level [lstchain.calib.camera.calib]
```

Reply:
> If you have the threshold set to `4094`, then it means that the threshold is not properly applied.
> 
> For the muon script, I do not think it is a good idea to have a duplicated calibration for the regular data analysis and muon data analysis, therefore I expect the script `lstchain_data_muon_analysis.py` to be deprecated and run the analysis only on calibrated files. I'll mark this in a comment on the script as no-gain selection, but I'm not sure if it is worth it to solve it at the moment.


The config files I am using indicate:
 ```
"gain_selector": "ThresholdGainSelector",
  "gain_selector_config": {
    "threshold":  4094
  }
}
```

Reply:
So what threshold value should we use then?

Reply:
then I think it will be affected... According to #200, the threshold should be 20.9
@vuillaut can you confirm?

Reply:
For *MC* yes.